### PR TITLE
PRO-395 Remove pre-selected ETH and PLR tokens

### PR DESCRIPTION
**Pull request by hemadri-tank-bacancy**
Remove pre selected token in Swap, Cross-chain and Send.

https://user-images.githubusercontent.com/78720269/183415546-1e15a772-74bc-477d-86ac-09e27a450d5d.mp4

### CODE REVIEW

File: src/hooks/transactions.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ export function useTransactionsEstimate(
 
 type UseTransactionFeeCheckResult = {|
   isEnoughForFee: boolean,
-  errorMessage?: string,
+    errorMessage?: string,
```
Comment (IAmKio) [inline src/hooks/transactions.js:95]: Is that spacing supposed to be there?

File: src/screens/Bridge/Exchange-CrossChain/CrossChain.tsx [hunk 2]
```diff
@@ -179,7 +174,7 @@ function CrossChain({ fetchCrossChainTitle }: Props) {
     <SendHighGasModal value={fromValue} contact={fromAddress} chain={chain} txFeeInfo={feeInfo} />
   ) : null;
 
-  const errorMessage = estimationErrorMessage | notEnoughForFeeErrorMessage;
+  const errorMessage = estimationErrorMessage | transactionFee?.errorMessage;
```
Comment (IAmKio) [inline src/screens/Bridge/Exchange-CrossChain/CrossChain.tsx:177]: Do you know if this returns a friendly error message or are we just returning the message from the server here?

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few questions but otherwise looks good.

File: src/hooks/transactions.js (hunk 1 above)
Comment (hemadri-tank-bacancy) [reply src/hooks/transactions.js:95]: I'll fix it.

Comment (hemadri-tank-bacancy) [reply src/hooks/transactions.js]: Fixed it.

**Review by IAmKio — APPROVED** [review]
LGTM

**Pull request merged**